services/QuillLMS/client/app/bundles/Teacher/components/general_components/tooltip/activity_details.jsx

```diff
@@ -39,19 +39,21 @@ export const ActivityDetails = ({ data }) => {
           dateBody = updated
         }
 
-      } else {
-        dateTitle = 'Due'
-        dateBody = firstCr.due_date
       }
     }
 
+    const publishDateForDisplay = publishDate || unitActivityCreatedAt
     const objectiveSection = activity_description ? <p><strong>Objectives:</strong>{` ${activity_description}`}</p> : <span />
-    const dateSection = dateTitle ? <p><strong>{`${dateTitle}: `}</strong>{`${moment(dateBody).format('MMMM D, YYYY')}`}</p> : <span />
-    const completedSection = completedTitle ? <p><strong>{`${completedTitle}: `}</strong>{`${moment(completedBody).format('MMMM D, YYYY')}`}</p> : <span />
+    const publishDateSection = <p><strong>{scheduled ? 'Scheduled For: ' : 'Published: '}</strong>{`${moment.utc(publishDateForDisplay).format('MMMM D, YYYY [at] h:mm a')}`}</p>
+    const dueDateSection = <p><strong>Due: </strong>{dueDate ? `${moment.utc(dueDate).format('MMMM D, YYYY [at] h:mm a')}` : 'N/A'}</p>
+    const dateSection = dateTitle ? <p><strong>{`${dateTitle}: `}</strong>{`${moment.utc(dateBody).format('MMMM D, YYYY')}`}</p> : <span />
+    const completedSection = completedTitle ? <p><strong>{`${completedTitle}: `}</strong>{`${moment.utc(completedBody).format('MMMM D, YYYY')}`}</p> : <span />
```

Comment:
The backend sends up timezone-agnostic times (because that's what's in the database). If you don't convert them to UTC, they are automatically converted to the browser time (which may or may not be what the user has saved as their timezone in the backend). So the coercion here is really just saying "don't convert me" - because what the backend is sending up is the correct time based on the user's timezone.